Frontend/insight/src/components/SampleClient.js

```diff
@@ -49,5 +52,5 @@ export default function SampleClient(props) {
         return () => clearInterval(interval);
     });
 
-    return <div></div>;
+    return <div><CardStack response={resp} /></div>;
```

Comment:
Might want to change this back to be on the main app page eventually but this works for now